splunk_connect_for_snmp_poller/manager/tasks.py

```diff
@@ -173,7 +174,7 @@ async def snmp_polling_async(
                 host,
                 ir.profile,
             )
-            mib_profile = server_config["profiles"].get(ir.profile, None)
+            mib_profile = profiles["profiles"].get(ir.profile, None)
```

Comment:
profile can be none if snmp_polling is called from poller_utilities. Will it not fail here trying to get profiles from None?